ui/app/AppLayouts/Profile/views/SettingsContentBase.qml

```diff
@@ -133,11 +138,13 @@ Item {
     SettingsDirtyToastMessage {
         id: settingsDirtyToastMessage
         anchors.bottom: scrollView.bottom
+        anchors.bottomMargin: root.ignoreDirty ? 40 : 0
```

Comment:
Looks like a suspicious random number to me 🤔

Reply:
![image](https://github.com/status-im/status-desktop/assets/5377645/63278d88-1a82-4b91-bc26-6618a18379c3)

Yeah right :)